### DISCUSSION TITLE
Auth - Feature Requests

### DISCUSSION THREAD

**Discussion by haritvsmurali** [ente-io/ente#2797] [Enhancements]
1. Browser Extensions - Chrome, Firefox, Safari.
2. Add Code - Upload Screenshot with QR code to import. 
3. Manually Set Logos - to upload image and set logos. 
4. Trash Feature - Deleted Codes will stay in bin for 30 days.
5. Wear & Watch OS Apps.

**Comment by sv3nnie** [comment]
Duplicates:
**1** Browser Extensions - https://github.com/ente-io/ente/discussions/1041
**3** Manually Set Logos - https://github.com/ente-io/ente/discussions/1036
**4** Trash Feature - https://github.com/ente-io/ente/discussions/492
**5** Wear & Watch OS Apps - https://github.com/ente-io/ente/discussions/485

**Comment by ua741** [comment]
As there are already existing open Discussion thread for all these requests, closing this.
